README.md

```diff
@@ -11,7 +11,7 @@ We understand that your time is valuable, and in anyone's busy schedule solving
 **Treat it like production code**. That is, develop your software in the same way that you would for any code that is intended to be deployed to production. These may be toy exercises, but we really would like to get an idea of how you build code on a day-to-day basis.
 
 ## How to submit?
-You can do this however you see fit - you can email us a tarball, a pointer to download your code from somewhere or just a link to a source control repository. Make sure your submission includes a small **README**, documenting any assumptions, simplifications and/or choices you made, as well as a short description of how to run the code and/or tests. Finally, to help us review your code, please split your commit history in sensible chunks (at least separate the initial provided code from your personal additions).
+Please upload your solution to a version control system hosting service, preferably [Github](https://www.github.com) or [Gitlab](https://www.gitlab.com), that allows code viewing and checkout without requiring an account. Afterward, email us a link to the branch that contains your solution. Make sure your submission includes a small **README**, documenting any assumptions, simplifications and/or choices you made, as well as a short description of how to run the code and/or tests. Finally, to help us review your code, please split your commit history in sensible chunks (at least separate the initial provided code from your personal additions).
```

Comment:
Should we only clarify that we will only review the branch sent to us and are unlikely to review other branches?

Reply:
I think there is no reason to be verbose, we only faced this issue once.